daml-lf/validation/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ da_scala_library(
     visibility = [
         "//compiler/scenario-service:__subpackages__",
         "//daml-lf:__subpackages__",
+        "//ledger/participant-integration-api:__subpackages__",
```

Comment:
Needed in `RejectionGenerators` because of exposed validation errors.

Reply:
fine with me.